README.rst

```diff
@@ -26,14 +31,35 @@ Initialize the database using::
 
     $ ./manage.py syncdb --noinput --migrate
 
-Then run the server usinng::
+Then run the server using (runs on http://localhost:8000)::
 
     $ ./manage.py runserver
 
+In order to access the admin interface, you'll need to create a super user::
+
+    $ ./manage.py createsuperuser
+    Then navigate to http://localhost:8000/admin/ once the server is up and running
```

Comment:
You probably want to remove this indent, otherwise it'll get the same `code` formatting.